### PR TITLE
Update And Add Workflows for Make Windows Binaries

### PR DESCRIPTION
**Pull request by SriMethan**

### CODE REVIEW

**Comment by ddugovic** [comment]
I don't know if this helps, but maybe also review the Appveyor configuration.  Again, I'm not a windows expert https://ci.appveyor.com/project/ddugovic/Stockfish/branch/master

File: .github/workflows/make.yml [hunk 1]
```diff
@@ -18,7 +18,11 @@ jobs:
         python-version: 3.7
     - name: configure
       run: sudo apt install expect && pip3 install python-chess && git clone https://github.com/ddugovic/chess-artist.git ./tests/chess-artist --depth 1
-    - name: make
+    - name: make default
```
Comment (ddugovic) [inline .github/workflows/make.yml:21]: I'm not excited about deviating too much from https://github.com/official-stockfish/Stockfish/tree/master/.github/workflows

Comment (SriMethan) [reply .github/workflows/make.yml:21]: I mean this Pr allows to download latest Windows Binaries from workflow example like 
https://github.com/SriMethan/Stockfish/actions/runs/1265045451 https://github.com/SriMethan/Stockfish/suites/3850374647/artifacts/95544413

**Comment by SriMethan** [comment]
@ddugovic Can you approve run workflows in this pr

File: .github/workflows/make.yml (hunk 1 above)
Comment (ddugovic) [reply .github/workflows/make.yml:21]: I guess I'll first sync my repository with `official-stockfish/master` and then see what the differences are.

Comment (SriMethan) [reply .github/workflows/make.yml:21]: Ok so should I close this pr or just keep it?

Comment (ddugovic) [reply .github/workflows/make.yml:21]: I guess keep it, there's no hurry.

Comment (ddugovic) [reply .github/workflows/make.yml:21]: I merged `official-stockfish/master` into my branch.  If you can fix merge conflicts and if the build succeeds I'll accept this PR.

Comment (SriMethan) [reply .github/workflows/make.yml:21]: Ok I will send a new Pr for this

Comment (SriMethan) [reply .github/workflows/make.yml:21]: @ddugovic this is the new pr https://github.com/ddugovic/Stockfish/pull/594

**Pull request closed without merging**